main.py

```diff
@@ -0,0 +1,423 @@
+from fastapi import FastAPI, HTTPException, Path, Body
+from fastapi.encoders import jsonable_encoder
+from typing import Annotated
+from models.Dispatcher import Dispatcher, UpdateDispatcherModel
+from models.Repair import RepairModel
+from models.Trailer import Trailer, UpdateTrailerModel
+from models.Truck import Truck, UpdateTruckModel
+from models.Manager import Manager, UpdateManagerModel
+from get_db_pymongo import get_database
+from bson import ObjectId
+from models.TruckDriver import TruckDriver, UpdateTruckDriverModel
+from utils import serialize_collection
+
+app = FastAPI()
+
+db = get_database()
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
+
+@app.get("/trucks/")
+async def get_trucks():
+    try:
+        collection = db.get_collection("trucks")
+        trucks = list(collection.find())
+        
+        return {"serialized_trucks": serialize_collection(trucks)}
+    except Exception as e:
+        print("An error occurred:", str(e))
+
+@app.get("/trucks/{truck_id}")
+async def get_truck(
+    truck_id: Annotated[str, Path(title= "id of object to get")]
+):
+    try:
+        collection = db.get_collection("trucks")
+        truck = collection.find_one({"_id": ObjectId(truck_id)})
+        
+        return {"serialized_truck": serialize_collection(truck)}
+    except Exception as e:
+        print("An error occurred:", str(e))
+
+@app.post("/trucks/", response_description="Add new truck", response_model=Truck)
+async def create_truck(truck: Truck = Body(...)):
+    try:
+        truck = jsonable_encoder(truck)
+        if "_id" in truck:
+            del truck["_id"]
+
+        collection = db.get_collection("trucks")
+
+        new_truck = collection.insert_one(truck)
+        created_truck = collection.find_one({"_id": ObjectId(new_truck.inserted_id)})
+        serialized_truck = serialize_collection(created_truck)
+        print({"serialized_truck": serialized_truck})
+        return serialized_truck
```

Comment:
This is too big for controller. You should create helper functions for each service, and have the logic in that function. Do this for all